src/native/fs/fs_napi.cpp

```diff
@@ -244,15 +244,21 @@ struct Stat : public FSWorker
     std::string _path;
     struct stat _stat_res;
 
+    bool _follow_symlinks;
+
     Stat(const Napi::CallbackInfo& info)
         : FSWorker(info)
+        , _follow_symlinks(true)
     {
         _path = info[1].As<Napi::String>();
         Begin(XSTR() << "Stat " << DVAL(_path));
+
+        Napi::Object config = info[0].As<Napi::Object>();
+        if (config.Has("follow_symlinks")) _follow_symlinks = config.Get("follow_symlinks").ToBoolean();
     }
     virtual void Work()
     {
-        int r = stat(_path.c_str(), &_stat_res);
+        int r = _follow_symlinks ? stat(_path.c_str(), &_stat_res) : lstat(_path.c_str(), &_stat_res);
```

Comment:
Why do we want to do it in the native? 
We have stat and `lstat` as ops, looks weird to call a `stat` op and get `lstat` 
We can do it on `namespace_fs` instead of here

Reply:
Then you have to modify all the stat calls in namespace_fs. now, it's exactly like in os. when you run stat <filename>, actually os runs lstat(2). just when you run stat -L, os runs stat(2)

Reply:
As far as I can tell, ` stat -L` is not running `lstat` op. 
Please run on a system `strace stat -L` and see if it call the `lstat` op. 

I don't see a reason why we should not move the logic into the `namespace_fs` and keep it mixed in the native (mixed as a single op is calling 2 different calls.)
@guymguym WDYT?

Reply:
But `stat` does run `lstat`